ui/static/ui/js/learning_resources.jsx

```diff
@@ -62,7 +62,10 @@ define('learning_resources', [
 
       var slug = this.props.selectedVocabulary.slug;
 
-      return <div className="col-sm-6">
+      return <div className="form-group">
+        <label htmlFor="vocabChooser" className="col-sm-4 control-label">
```

Comment:
It uses the JSX spread operator so any attributes you define will get passed down, including the `id` attribute